### ISSUE TITLE
Weekly Classes Blueprint

### ISSUE THREAD

**Issue by liamphan** [PSUGraphicDesign/colevity#4]
_Weekly Classes_ page should include the following editable fields. The data for the events will most likely be consistent through the coming weeks, but the data should be editable in the case that an event may change dates, time, or any other field of information.
- [x] Title
- [x] Time
- [x] Days
- [x] Description
- [x] Ability Level
- [x] Age Range
- [x] Pricing
- [ ] Space Availability
- [x] Reservations (Through Mindbody?)
- [ ] Location
- [x] Instructor
- [ ] Photography

**Comment by jeburchett** [comment]
I claim this one!

**Comment by jeburchett** [comment]
I decided to leave out space availability and location. The availability will probably be tracked via MindBody and location might end up being repetitive. I also left out the photo for now... The day field for the classes looks weird because Kirby broke it into two columns and I can't find a way for it to be just one column. Or if that's even a thing.

For the classes page, I copied and pasted it from performances and right now it has "num: date" which I'm assuming is incorrect but I don't really know what to actually put there so I left if for now.
